web/app/middleware.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
```

Comment:
Can we make TunnelV2 a separate app, and deploy it to a different container? That will eliminate the need for this middleware, or other places where we use if/else to switch between tunnel v1/v2.